schemas/alternate-registration-schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A representation of multiple registrations for the same entity",
+  "type": "object",
+  "properties": {
+    "data_type": {
+      "enum": [
+        "alternate-registration"
+      ]
+    },
+    "alternate_entities": {
+      "description": "TBC",
+      "type": "array",
+      "items": {
+        "$ref": "includes/entity.json",
+        "minLength": 2
+      }
+    },
+    "alternate_registration_start_date": {
+      "description": "date when alternate registration started",
+      "type": "string",
+      "format": "date"
+    },
+    "alternate_registration_end_date": {
+      "description": "date when alternate registration ended",
+      "type": "string",
+      "format": "date"
+    },
+    "publication_date": {
+      "description": "TBC",
```

Comment:
"the publication date of the filing/notice that gives details of the alternate registration"

Reply:
✅